framework/interface/ui_handlers.py

```diff
@@ -440,11 +441,12 @@ class FileRedirectHandler(custom_handlers.UIRequestHandler):
     SUPPORTED_METHODS = ('GET')
 
     def get(self, file_url):
+	file_url = file_url[1:]
```

Comment:
I have already checked them on Gvim, Vim, gedit and leafpad, they were not showing any indentation problem but just now I checked it on sublime text 3 and it shows the indentation problem :P